armotypes/vulnerabilitytypes.go

```diff
@@ -81,6 +81,8 @@ type Vulnerability struct {
 	CvssInfo       []CvssInfo                 `json:"cvssInfo"`
 	EpssInfo       EpssInfo                   `json:"epssInfo"`
 	CisaKevInfo    CisaKevInfo                `json:"cisaKevInfo"`
+	BaseScore      float64                    `json:"baseScore"`
```

Comment:
add it as part of cvss info, extend the info to include both array of all cvss and these 2 feilds